### ISSUE TITLE
{Feature}: On Mobile, Insufficient contrast of "You are needed" tag + text, design, small issue

### ISSUE THREAD

**Issue by AVbiser** [zetkin/app.zetkin.org#2732]
**PROBLEM:**
1. The orange **"You are needed"** tags have relatively low contrast against the white background, potentially failing WCAG 2.1 contrast requirements + making it difficult for users with low vision or color blindness to read.
2. Additionally, there is insufficient contrast between the tag fill itself (#FEE5B1) and the text in the tag (#F47C0D).

<img width="422" alt="Image" src="https://github.com/user-attachments/assets/c30d496f-fa83-485c-884d-56a5d16301a7" />

<img width="206" alt="Image" src="https://github.com/user-attachments/assets/4b10c5cb-de25-466b-b32d-bebb2aa94184" />

**SOLUTION:**
1. Increase the contrast of the white background and the tag color by using a darker, more saturated orange or a different color altogether on the tag fill that meets the minimum contrast ratio of 4.5:1 for normal text and 3:1 for large text. Alternatively, add a border or background shading to improve visibility.
2. Change the color of the text to a color that will have at least 4.5 : 1 contrast to the tag background in order to meet the WCAG 2.1 contrast requirements.

Example solution:

<img width="747" alt="Image" src="https://github.com/user-attachments/assets/ed77258e-6883-4e3f-a4d7-0f2ab4a40ae3" />

**Comment by richardolsson** [comment]
This is based on the figma mock-ups and not the actual implementation. Will close now, and we might revisit if the problem also exists in the actual UI.
